src/Core/src/Core/IBorder.cs

```diff
@@ -3,6 +3,7 @@
 /// <summary>
 /// Provides functionality to provide a border.
 /// </summary>
+[System.Obsolete("IBorder is not used and will be removed in a future release. Use IBorderView or IBorderStroke instead.")]
```

Comment:
Would be a good idea to hide it from users with `[EditorBrowsable(EditorBrowsableState.Never)]`?